setup.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+# This script sets up a Ubuntu machine to run this Node.js app locally
+
+# Install Node.js runtime environment
+if [ -n "$(nodejs --version)" ]
+then
+  echo "Node.js is already installed"
+else
+  echo "Installing nodejs"
+  sudo apt update
+  sudo apt install -y nodejs
+fi
+
+# Install npm package manager for JS
+if [ -n "$(npm --version)" ]
+then
+  echo "npm is already installed"
+else
+  echo "Installing npm"
+  sudo apt update
+  sudo apt install -y npm
+fi
+
+# Install git
+if [ -n "$(git --version)" ]
+then
+  echo "git is already installed"
+else
+  echo "Installing git"
+  sudo apt-get update
+  sudo apt-get install -y git-all
+fi
+
+# Clone the repository
+if [ -d ./relativepath.tech ]
```

Comment:
I think we should assume this scripts runs from the repo and is already checked out. There is another "outer context" of "onboarding a new developer" that could be where we do git checkouts, but for this context assume we start at "already checked out and from this repo run..."

Reply:
I understand Chris' attempt to make a script that can be downloaded without the need for cloning the repo, and can setup the project regardless of the state of the user's machine. We have discussed this and gone another route.